ReSharper.FSharp/src/FSharp.Psi.Services/src/Generate/GenerateProvider.fs

```diff
@@ -323,7 +330,13 @@ type FSharpOverridingMembersBuilder() =
         | :? IUnionRepresentation as unionRepr ->
             unionRepr.UnionCasesEnumerable
             |> Seq.tryHead
-            |> Option.iter EnumCaseLikeDeclarationUtil.addBarIfNeeded 
+            |> Option.iter EnumCaseLikeDeclarationUtil.addBarIfNeeded
+        | :? ITypeAbbreviationRepresentation as abbrRepr ->
+            if isNotNull abbrRepr.FirstChild then
+                addNodesBefore abbrRepr.FirstChild [
+                    FSharpTokenType.BAR.CreateLeafElement()
+                    Whitespace()
+                ] |> ignore
```

Comment:
@dawedawe Sorry, it seems this change is incorrect. Have you checked the parsed tree for the following code?

```fsharp
type SingleCaseUnion = | Case
```